docs/content/how-to/agent/create-agent-cluster.md

```diff
@@ -398,9 +608,142 @@ spec:
+clusteroperator.config.openshift.io/kube-apiserver                             4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-controller-manager                    4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-scheduler                             4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-storage-version-migrator              4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/monitoring                                 4.11.5    True        False         False      21m     
+clusteroperator.config.openshift.io/network                                    4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/openshift-apiserver                        4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/openshift-controller-manager               4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/openshift-samples                          4.11.5    True        False         False      23m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager                 4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager-catalog         4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager-packageserver   4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/service-ca                                 4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/storage                                    4.11.5    True        False         False      25m     
+~~~
+
+## Enabling Node Auto-Scaling for the Hosted Cluster
+
+Auto-scaling can be enabled, if we enable auto-scaling when more capacity is required in our Hosted Cluster a new Agent will be installed (providing that we have spare agents). In order to enable auto-scaling we can run the following command:
+
+> **INFO:** In this case the minimum nodes will be 2 and the maximum 5.
+
+~~~sh
+oc -n ${CLUSTERS_NAMESPACE} patch nodepool ${HOSTED_CLUSTER_NAME} --type=json -p '[{"op": "remove", "path": "/spec/replicas"},{"op":"add", "path": "/spec/autoScaling", "value": { "max": 5, "min": 2 }}]'
+~~~
 
-Configure wildcard DNS A record or CNAME that references the LoadBalancer Service's external IP.
-Configure a wildcard *.apps.<cluster_name>.<base_domain>. DNS entry referencing the IP stored in
-$INGRESS_IP that is routable both internally and externally to the cluster.
-No newline at end of file
\ No newline at end of file
+When the capacity is not required anymore, after 10 minutes the agent will be decommissioned and placed in the spare queue again.
```

Comment:
If 10 minutes passes without requiring the additional capacity the agent

Reply:
Fixed.